package/MDAnalysis/analysis/pcz.py

```diff
@@ -0,0 +1,481 @@
+"""
+A PCA module for MDAnalysis
+==========================================================================
+
+:Authors: The pyPcazip Developers - Ardita Shkurti, Ramon Goni, Pau Andrio, Elena Breitmoser, Iain
+Bethune, Modesto Orozco, Charles Laughton
+:Year: 2015
+:Copyright: BSD
```

Comment:
If you can relicense then that would indeed make life easier but BSD-3 clause is compatible with GPL v2.

However, in this case the license should be added to the LICENSE file and saying that "Module MDAnalysis.analysis.pcz is licensed under the following terms: ..."